ttt/lib/ttt/board.ex

```diff
@@ -12,30 +16,138 @@ defmodule TTT.Board do
     |> mark_with_fewest_moves
   end
 
-  def get_move_count_per_mark(board, marks) do
+  def game_over?(board) do
+    Enum.any?([not remaining_spaces?(board), found_winner?(board)], is_true?)
+  end
+
+  def winning_mark(board) do
+    winning_mark_in_row(board) <>
+     winning_mark_in_column(board) <>
+      winning_mark_in_diagonal(board)
+  end
+
+  def found_winner?(board) do
+    Enum.any?(win_in_any_direction?(board), is_true?)
+  end
+
+  defp win_in_any_direction?(board) do
+    [column_win?(board), row_win?(board), diagonal_win?(board)]
```

Comment:
This could simply be 
`column_win?(board) or row_win?(board) or diagonal_win?(board)`.
then you don't need to loop over it with `any?(..., is_true?)`